spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/TaskConfigurationProperties.java

```diff
@@ -41,11 +41,22 @@
 	@NotBlank
 	private String composedTaskRunnerName = "composed-task-runner";
 
+	@NotBlank
+	private String schedulerTaskLauncher = "scheduler-task-launcher";
```

Comment:
The name `schedulerTaskLauncher` implies the launcher itself instead of being the `name` of the launcher. Can we rename this to be something similar to `schedulerTaskLauncherName`?

Reply:
cc: @cppwfs

Reply:
@ilayaperumalg  Fer sure!

Reply:
Done